### PR TITLE
Fixes #513 move observer tex builder. Also fixes #512

### LINKED ISSUE #512: AbstractCommandCollectorPresenter: AddCommand with Func

### LINKED ISSUE #513: Move ObserverBuilder TexReporter from MoBi to Core

### PR DESCRIPTION
**Pull request by msevestre**

### CODE REVIEW

**Comment by msevestre** [comment]
Moved from MoBi to core. Maybe some builders are missing. We'll have to try. But it needs to be merged for that unfrotunatley

**Comment by msevestre** [comment]
Moved from MoBi to core. Maybe some builders are missing. We'll have to try. But it needs to be merged for that unfrotunatley

File: src/OSPSuite.Infrastructure/Reporting/TeXBuilder/ObserverBuilderTeXBuilder.cs [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+            listToReport.Add(string.Format(PROPERTY_PROMPT_FORMAT, TYPE, Captions.MoleculeObserver));
+         if (containerObserver != null)
+            listToReport.Add(string.Format(PROPERTY_PROMPT_FORMAT, TYPE, Captions.ContainerObserver));
+
+         listToReport.Add(string.Format(PROPERTY_PROMPT_FORMAT, DIMENSION, observerBuilder.Dimension));
+
+         if (observerBuilder.ForAll)
+         {
+            listToReport.Add(string.Format(PROPERTY_PROMPT_FORMAT, CALCULATED_FOR_ALL_MOLECULES,
+               observerBuilder.ForAll));
```
Comment (Yuri05) [inline src/OSPSuite.Infrastructure/Reporting/TeXBuilder/ObserverBuilderTeXBuilder.cs]: if the molecules Exclude-List is not empty - it must be added here to the TEX-output

Comment (msevestre) [reply src/OSPSuite.Infrastructure/Reporting/TeXBuilder/ObserverBuilderTeXBuilder.cs]: This comes from MoBi. Copied as is. Please create another issue if this needs to be changed

Comment (Yuri05) [reply src/OSPSuite.Infrastructure/Reporting/TeXBuilder/ObserverBuilderTeXBuilder.cs]: Issue created: #517

**Pull request merged**